code/modules/mob/living/simple_animal/slime/powers.dm

```diff
@@ -48,31 +48,55 @@
 	var/mob/living/simple_animal/slime/S = owner
 	S.Feed()
 
-/mob/living/simple_animal/slime/proc/CanFeedon(mob/living/M)
+/mob/living/simple_animal/slime/proc/CanFeedon(mob/living/M, silent = FALSE)
 	if(!Adjacent(M))
 		return 0
 
 	if(buckled)
 		Feedstop()
 		return 0
 
+	if(issilicon(M))
+		return 0
```

Comment:
Yes I know the rest of the proc does it wrong but that just means you need to fix it if you're touching related code
`return FALSE`

Reply:
Ack, I hadn't even realized what it was doing. That's gross, I'm fixing the whole proc.